js/apps/admin-ui/maven-resources/theme/phasetwo.v2/admin/messages/messages_en.properties

```diff
@@ -3027,11 +3028,16 @@ secondaryColor800Help=Color choice for the palette used in the Admin Portal. Fol
 secondaryColor900Help=Color choice for the palette used in the Admin Portal. Follows Tailwind CSS color naming conventions. 100 being the lightest shade and 900 being the darkest shade.
 cssHelp=Custom CSS to be applied. Ensure CSS is valid as no validation is performed.
 colorHelpInvalid=Color must be a valid hex value.
-noIDPsAvailable=No IdPs are available to assign to this Org. Add an Identity Provider to be able to assign to this Organization. 
+noIDPsAvailable=No Identity Providers are available to assign to an Organization. Add an Identity Provider to the Realm. 
 noIDPAssigned=No Identity Providers are assigned to this Organization. Assign an Identity Provider to be able to use it with this Organization.
-idpAssignedToOrg=Identity Provider assigned to Organization
+idpAssignedToOrg=Identity Provider Assigned to Organization
 idpChangeHelpText=Change the active IdP for this Organization.
 idpAssign=Assign Identity Provider
```

Comment:
Can just be "Assign"

Reply:
this is used for the title, there's a idpAssign variable for the action, but that was a duplicate just being "Assign" so I removed it and used the original